src/mscorlib/corefx/System/Globalization/CultureData.Windows.cs

```diff
@@ -607,5 +607,26 @@ private static CultureInfo[] EnumCultures(CultureTypes types)
         {
             throw new NotImplementedException();
         }
+
+        private static string GetConsoleFallbackName(string cultureName)
+        {
+            throw new NotImplementedException();
+        }
+
+        private bool IsSupplementalCustomCulture
+        {
+            get { throw new NotImplementedException(); }
+        }
+        
+        private bool IsWin32Installed
+        {
+            get { throw new NotImplementedException(); }
+        }
+        
+        private bool IsReplacementCulture
+        {
+            get { throw new NotImplementedException(); }
```

Comment:
> The current Windows implementation under corefx is not used but we have it to make it in sync with corert

Ah, I see, ok.  As long as there's a comment somewhere in the file indicating that it's not used and there are lots of missing pieces, that's fine.